buku

```diff
@@ -1456,17 +1464,18 @@ class BukuDb:
             if delay_delete:
                 self.conn.commit()
             else:
-                self.commit_delete()
+                self.commit_delete(retain_order=retain_order)
```

Comment:
In most cases, this value is passed directly as the respective parameter of calls which may result in record deletion